src/Nethermind/Nethermind.Evm/ExecutionEnvironment.cs

```diff
@@ -77,5 +105,7 @@ public ExecutionEnvironment
 
         /// <example>If we call TX -> DELEGATECALL -> CALL -> STATICCALL then the call depth would be 3.</example>
         public readonly int CallDepth;
+
+        public readonly IEnumerable<Address> AuthorizedAddresses;
```

Comment:
never used?

Reply:
Currently Authority counts as warmed up, and most likely the code source addresses will be added as well. 
This is used in the `WarmUp` method.

Reply:
No it isn't: `env.TxExecutionContext.AuthorizedCode.AuthorizedAddresses`

Reply:
Oh yea, was thinking of the other contexts